cardano-api/src/Cardano/Api/ScriptData.hs

```diff
@@ -3,10 +3,17 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE InstanceSigs #-}
```

Comment:
I think HLS inserted this for me and it isn't needed. I'll remove it when I next push